support/doc/production.md

```diff
@@ -182,8 +198,19 @@ server {
 }
 ```
 
+To generate the certificate for your domain as required to make https work, you have two alternatives:
+
+```
+$ sudo certbot --authenticator standalone certonly -d peertube.example.com && openssl dhparam -out /etc/nginx/dhparam.pem 4096
+```
+
+```
+$ sudo certbot --authenticator standalone --installer nginx --pre-hook "systemctl stop nginx" --post-hook "systemctl start nginx"
```

Comment:
`post-hook "systemctl reload nginx"` should be enough

Reply:
do `nginx -t && systemctl reload nginx` so that it won't reload the server config if it failed to apply it.